tests/components/yandex_transport/test_yandex_transport_sensor.py

```diff
@@ -101,17 +77,12 @@ def true_filter(reply, filter_routes=None):
     """Return the contents of _state."""
     await assert_setup_sensor(hass, TEST_CONFIG)
     state = hass.states.get("sensor.test_name")
-    assert state.state == dt_util.utc_from_timestamp(1568659253).isoformat(
-        timespec="seconds"
-    )
+    assert state.state == RESULT_STATE
 
 
 async def test_filtered_attributes(hass, mock_requester):
     """Return the contents of attributes."""
     await assert_setup_sensor(hass, TEST_CONFIG)
     state = hass.states.get("sensor.test_name")
-
-    true_attrs, true_state = true_filter(REPLY, filter_routes=ROUTES)
-    assert state.state == true_state
-    state_attrs = {key: state.attributes[key] for key in true_attrs}
-    assert state_attrs == true_attrs
+    state_attrs = {key: state.attributes[key] for key in FILTERED_ATTRS}
+    assert dict(state_attrs) == FILTERED_ATTRS
```

Comment:
We don't need to copy to another dict. We create a new dict just above in the dict comprehension.